src/modules/display.js

```diff
@@ -0,0 +1,68 @@
+// eslint-disable-next-line
+import { getAllMovies, getAllLikes } from './service.js';
+
+export const Display = async () => {
+  const shows = await getAllMovies();
+  const likes = await getAllLikes();
+
+  const showLikes = (showId) => {
+    let totalLikes = '';
+
+    const like = likes.filter((l) => l.item_id === showId);
+    if (like[0]) {
+      totalLikes = like[0].likes.toString();
+    }
+    return totalLikes;
+  };
+
+  let htmlshow = '';
+
+  shows.forEach((show) => {
+    htmlshow += `
+  <div class="show-cards">
+    <img src="${show.show.image.medium}" alt="show Images">
+        <div class="details">
+            <h2>${show.show.name} 
+              <button class="btn" id="btnLikes"><i class="bi bi-heart" data-showId="${
+  show.show.id
+}"></i></button></h2>
+              <p id="likeId" >Likes "${showLikes(show.show.id)}"</p>
```

Comment:
After clicking on the like button is necessary to refresh the browser to see the total number of likes. I suggest that after clicking the button, the number of likes is dynamically updated, this will make the system more user-friendly.